charts/fybrik/files/taxonomy/datacatalog.json

```diff
@@ -172,6 +172,41 @@
           "description": "Tags associated with the asset"
         }
       }
+    },
+    "UpdateAssetRequest": {
+      "type": "object",
+      "required": [
+        "assetID",
+        "resourceMetadata",
+        "details"
+      ],
+      "properties": {
+        "assetID": {
+          "$ref": "taxonomy.json#/definitions/AssetID",
+          "description": "The destination catalog id in which the asset will be updated based on the information provided in ResourceMetadata and ResourceDetails field Asset ID of the to-be updated asset"
+        },
+        "credentials": {
+          "description": "The vault plugin path where the destination data credentials will be stored as kubernetes secrets",
+          "type": "string"
+        },
+        "details": {
+          "$ref": "#/definitions/ResourceDetails",
+          "description": "Source asset details like connection and data format"
```

Comment:
Source is confusing, rename to asset details...

Reply:
Updated. Thanks.